### ISSUE TITLE
mdeditor

### ISSUE THREAD

**Issue by 2060yxz** [icewind1991/files_markdown#83]
this is very good,if nextcloud use this deitor.   https://www.zybuluo.com/mdeditor#

**Comment by noseshimself** [comment]
> this is very good,if nextcloud use this editor. https://www.zybuluo.com/mdeditor#

Sorry, but many of us have problems finding a download link in this (and of course understanding the explanations). Even if it sounds elitist: If it is not in English it won't do much for the people who have to implement it.

**Comment by bernd-wechner** [comment]
Agreed, it looks good, but alas I can't read Chinese and cannot see how to possibly install it on a nextcloud to replace this poorly maintained editor here.
